styles.css

```diff
@@ -33,7 +33,34 @@ img{
 }
 
 @media(max-width:600px){
-  .main{
-    background:red;
+  main{
+    flex-direction: column;
   }
+  header {
+    order:1;
+  }
+  main {
+    order: 2;
+  }
+  aside{
+    order: 3;
+  }
+}
+footer{
+  order:4;
+}
```

Comment:
You are applying to proper values to order, but make sure that your indentation is correct. It looks like `footer` is placed outside of the `@media` query, meaning that it would not be applied if viewing this page on a mobile device.